exe/backend/src/main.cc

```diff
@@ -107,6 +109,7 @@ int main(int argc, char const* argv[]) {
       ("vehicle_positions_url,v", bpo::value(&vehicle_position_url)->required(), "URL for vehicle positions")  //
       ("predictor,P", bpo::value(&predictor)->default_value("gtfs-position-tracker"), "Choose which predictor to use")
       ("in,i", bpo::value(&in)->required(), "input path")  //
+      ("protobuf-input,pi", bpo::value(&pin)->default_value("protobuf"), "input path")  //
```

Comment:
above we also used _ as seperator
```suggestion
      ("protobuf_input,pi", bpo::value(&pin)->default_value("protobuf"), "input path")  //
```